sql/core/src/test/resources/sql-tests/results/ansi/interval.sql.out

```diff
@@ -106,9 +112,15 @@ struct<(- INTERVAL '-1 months 1 days -1 seconds'):interval>
 -- !query
 select -interval -1 month 1 day -1 second
 -- !query schema
-struct<(- INTERVAL '-1 months 1 days -1 seconds'):interval>
```

Comment:
Is multi-value-unit interval itself non-ANSI, so why its result shall be an ANSI interval?

Reply:
In the future (I hope in the near future), we will get rid off `CalendarIntervalType`. If we are going to support multi-value-unit interval, we should be able to parse them to ANSI intervals.